vipps-login-api.md

```diff
@@ -220,7 +220,7 @@ Vipps Login currently supports the following scopes:
 | Scopes      | Description                                    | User consent required  |
 | ------------| -----------------------------------------------|-------- |
 | openid      | Scope used to request an Id-token. It provides the claim “sub” which is a unique id for the end user at that particular merchant. Note: Different merchants will get different subs for the same end user.              |   no    |
-| address     | List containing the users addresses. Will always contain home, but can also include work and other.    |   yes   |
+| address     | List containing the users addresses. Will always contain home, but can also include work and other.   The claim 'address' returns the address set as "default" for the Vipps user. The other addresses, if any, of the user are served under claim 'other_address' as a list. We recommend that merchants fetch all addresses on a user and allow the user to choose which address to use in the relevant context |   yes   |
```

Comment:
I missed this one when we where rewriting after changing the api. It's wrong to say that it is a list since it now are two claims whereas one of them is an object and the other is a list. I tried to rewrite it, but I'm not conviced it got that much better.
```suggestion
| address     | Users addresses given as claims 'address' and 'other_address'. The claim 'address' returns the address set as 'default' for the Vipps user. And the claim 'other_address' returns all other addresses of the end user, if any. One of the addresses will always be home, but can also include work and other, and would not necessary be marked as default. We recommend that merchants fetch all addresses on a user and allow the user to choose which address to use in the relevant context |   yes   |
```